doc_src/tutorial.rst

```diff
@@ -649,6 +649,23 @@ This prompt would look like:
     :purple:`02/06/13`
     :red:`/home/tutorial >` _
 
+You can use transient prompt functionality by setting ``fish_transient_prompt`` to 1 and checking for ``--final-rendering`` argument::
```

Comment:
This should explain what a "transient prompt" is.

Reply:
I found an example explanation at https://github.com/romkatv/powerlevel10k#transient-prompt

Reply:
I don't think that example helps a lot because it's pretty specific to p10k's use of it. How about something like

"You can make your prompt transient by setting fish_transient_prompt to 1. That means fish will redraw the prompt before running a commandline, so you can change it then. Usually this is used to trim it down to declutter your terminal's scrollback history. When it runs the prompt again, it will pass a ``--final-rendering`` argument, so use that to distinguish."